src/app/AppContainer.js

```diff
@@ -50,15 +61,24 @@ class AppContainer extends React.Component<PropsType> {
 		if ( this.props.showHtml ) {
 			this.updateHtmlAction( this.props.editedPostContent );
 		}
+
 		const html = serialize( this.props.getBlocks() );
-		RNReactNativeGutenbergBridge.provideToNative_Html( html, this.lastHtml !== html );
+		const hasChanges = this.state.title !== this.lastTitle || this.lastHtml !== html;
```

Comment:
Noticed that `hasChanges` is returned to the Android (wp-android) side as `false`, even if i changed the content of the post in the editor. The first time the native side calls the method to retrieve the content from RN, `true` is returned, and then the next calls to retrieve the content return `false`.

Reply:
Is this a regression or is this a problem with the original logic (before my changes) too?

The reason I'm asking is that the condition and logic haven't changed much in terms of what happens when you edit the content, and if this issue was part of the original logic, it's outside of the scope of what this PR tries to do.